gluster-exporter/main.go

```diff
@@ -3,27 +3,32 @@ package main
 import (
 	"flag"
 	"fmt"
+	"bufio"
 	"net/http"
 	"os"
 	"runtime"
 	"time"
-
+	"strings"
+	"regexp"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
+const (
+	defaultGd1Workdir = "/var/lib/glusterd"
+	defaultGd2Workdir = "/var/lib/glusterd2"
+)
 
 // ExporterVersion and GitSHA
 // These are set as flags during build time. The current values are just placeholders
 var (
 	ExporterVersion         = ""
 	GitSHA                  = ""
-	defaultGlusterd1Workdir = ""
```

Comment:
Retain these four variables as is. Values for these variables substituted during build. using `LDFLAGS`